test/irmin-pack/layered.ml

```diff
@@ -34,8 +34,8 @@ let fresh_name =
 let index_log_size = Some 4
 
 module Conf = struct
-  let entries = 32
-  let stable_hash = 256
+  include Irmin_tezos.Conf
```

Comment:
Couldn't it be better to include `Conf` here (as you declared `Common.Conf = Irmin_tezos.Conf` and you opened `Common` in this file)?
But maybe is it less explicit?

Reply:
Good idea! Done.